<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Themes</class>
 <widget class="qMRMLWidget" name="Themes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>446</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Color theme:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="ThemeComboBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="InvertCheckBox">
       <property name="text">
        <string>Invert secondary?</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>primaryColor</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>primaryLightColor</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>secondaryColor</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>secondaryLightColor</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>secondaryDarkColor</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>primaryTextColor</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>secondaryTextColor</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ctkColorPickerButton" name="primaryColorPickerButton"/>
     </item>
     <item row="4" column="1">
      <widget class="ctkColorPickerButton" name="primaryLightColorPickerButton"/>
     </item>
     <item row="5" column="1">
      <widget class="ctkColorPickerButton" name="secondaryColorPickerButton"/>
     </item>
     <item row="6" column="1">
      <widget class="ctkColorPickerButton" name="secondaryLightColorPickerButton"/>
     </item>
     <item row="7" column="1">
      <widget class="ctkColorPickerButton" name="secondaryDarkColorPickerButton"/>
     </item>
     <item row="8" column="1">
      <widget class="ctkColorPickerButton" name="primaryTextColorPickerButton"/>
     </item>
     <item row="9" column="1">
      <widget class="ctkColorPickerButton" name="secondaryTextColorPickerButton"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Style</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="StyleComboBox">
       <item>
        <property name="text">
         <string>Slicer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Material</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Load Theme</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="clearButton">
     <property name="text">
      <string>Clear Theme</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
